protobuf/aelf/core.proto

```diff
@@ -89,3 +87,18 @@ message SInt64Value
 {
     sint64 value = 1;
 }
+
+message MerklePath {
+    repeated MerklePathNode merklePathNodes = 1;
+}
+
+message MerklePathNode{
+    Hash hash = 1;
+    bool isLeftChildNode = 2;
+}
+
+message BinaryMerkleTree {
+    repeated Hash nodes = 1;
+    Hash root = 2;
```

Comment:
The root is already the last element in nodes, do we need to duplicate it in message? Why don't we delete it (root) from proto declaration add it in partial implementation (BinaryMerkleTree.cs) instead?

`public Hash Root => Nodes.Any() ? Nodes.Last() : Hash.Empty;`

Reply:
We should make sure all the merkle tree nodes have been computed completely , in which case the expression above is valid.